packages/cdc-react/vite.config.ts

```diff
@@ -15,6 +15,13 @@ export default defineConfig({
     environment: "jsdom",
     setupFiles: "./src/tests/setup.ts",
   },
+  css: {
+    preprocessorOptions: {
+      scss: {
+        includePaths: ["./node_modules/@uswds/uswds/packages"],
```

Comment:
I'm concerned that referencing the alias like this directly from `node_modules` may cause some side effects. It seems a bit unconventional.

Reply:
I also hate it.  Open to suggestions.

Reply:
Let's keep it this way. Bootstrap also has this setup. [Bootstrap & Vite](https://getbootstrap.com/docs/5.2/getting-started/vite/)